code/game/machinery/_machinery.dm

```diff
@@ -333,6 +333,7 @@
 
 
 /obj/machinery/can_interact(mob/user)
+	var/dextrous_cache = FALSE
```

Comment:
This isn't really a cache, it's just an upvalue.

Reply:
Just name it `is_dextrous`, and declare it near where you're actually setting it.

Reply:
done!